e2e/cypress/integration/notifications/browser_tab_notification_spec.js

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @notifications
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Notifications', () => {
+    let user1;
+    let user2;
+    let team2;
+    let testTeam1TownSquareUrl;
+    let testTeam2TownSquareUrl;
+
+    const timestamp = Date.now();
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            testTeam1TownSquareUrl = `/${team.name}/channels/town-square`;
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+
+            cy.apiCreateTeam(`test-team-${timestamp}`, `test-team-${timestamp}`).then(({team: anotherTeam}) => {
+                team2 = anotherTeam;
+                testTeam2TownSquareUrl = `/${team2.name}/channels/town-square`;
+                cy.apiAddUserToTeam(team2.id, user1.id);
+                cy.apiAddUserToTeam(team2.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T560_1 Browser tab and team sidebar unreads and mentions - Mention in different team', () => {
+        let originalFavicon = '';
+
+        // # User 1 view team A
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+        cy.wait(TIMEOUTS.FIVE_SEC);
+
+        // # Remove all unreads before test
+        cy.get('#publicChannelList').should('be.visible').get('li').eq(1).click();
+
+        // * Check for no unreads or mentions
+        cy.get('.unread-title').should('not.exist');
+
+        // # Save path of current favicon path
+        cy.get('link[rel=icon]').should('have.attr', 'href').then(($path) => {
+            originalFavicon = $path;
+        });
+
+        // # Have another user view team B
+        cy.apiLogin(user2);
+        cy.visit(testTeam2TownSquareUrl);
+        cy.wait(TIMEOUTS.FIVE_SEC);
+
+        // # Post a message with mentioning User 1 from Town Square
+        cy.postMessage(`@${user1.username}`);
+
+        // # Return to User 1
+        cy.visit(testTeam1TownSquareUrl);
+        cy.apiLogin(user1);
```

Comment:
This will not work as you'd expect. Proper way is to remain user1 on logged in state. You may use `cy.postMessageAs` for user2 without logging in, like
```suggestion
        // # Have the other user post an at-mention for you in any channel on team B
        cy.apiGetChannelByName(teamB.name, 'off-topic').then(({channel}) => {
            cy.postMessageAs({sender: user2, message: `@${user1.username}`, channelId: channel.id});
        });
```